### ISSUE TITLE
Migrate to material icons

### ISSUE THREAD

**Issue by torstendiesel** [MarcusWolschon/osmeditor4android#2991]
Vespucci could benefit from some new UI icons. The current ones look a bit dated, are inconsistent (For example, pressing the "tag" button and the "ABC" button sometimes takes you to the same place), and lacking accessibility-wise (For example, [this icon](https://github.com/MarcusWolschon/osmeditor4android/blob/master/svg/extract_node_light.svg)). 

To help with these issues, I would like to propose migrating to material icons. Actions which were previously unclear to me personally because of poor icon choice, specifically the icon for "enter address", would be made more clear:

![Image](https://github.com/user-attachments/assets/add05c8e-0049-4ce9-9602-2d9fd5a9796a) to ![Image](https://github.com/user-attachments/assets/58cd6a17-1c27-4f26-898f-38938db0d580), for instance. 

Not all icons in /svg can immediately be migrated over. I could not find an icon for "bridge" in the material library, but it appears in the code. 

Further examples: 

![Image](https://github.com/user-attachments/assets/8f5ae747-081b-40a6-bfe4-0bfa3d212c50): Follow line ("Airline stops")

![Image](https://github.com/user-attachments/assets/f1d63810-259b-4c49-be4e-5ab399873bae): Bug

![Image](https://github.com/user-attachments/assets/22001991-f95a-4f9e-90d7-f0c57091494a): Add note (or find a way to incorporate the same icon as iD for consistency)

![Image](https://github.com/user-attachments/assets/4c0b4f58-036c-415e-80bd-d7736906577d): Multi-select (Current one in use: ![Image](https://github.com/user-attachments/assets/1cfc9bca-9dc9-453b-9180-13f6f1b7a700)

Is this something worth pursuing?

**Comment by simonpoole** [comment]
Not Android Studio specific, simply jow I do it: to create a PR you typically fork (in github) the repo you want to work on to your github account. Then you push your local branch of the repo that you have been working on -to your- fork on github. If you then go to github online it should ask you if you want to create a PR.

**Comment by torstendiesel** [comment]
I see. Let me give it a try.

**Comment by torstendiesel** [comment]
Hi! Sorry this has taken a while, I'm a student and my semester got quite busy. Quite frankly, I forgot about this. I'd like to continue working on this, I'll let you know how it goes! (I still have all of my progress from before, so it should go pretty quickly -- I'll just make sure the app still builds, sync all the changes which happened in the meantime, and create a PR. I'll let you know if I run into issues with it.

**Comment by torstendiesel** [comment]
Alright, a draft of it is done! 

 - I renamed some of the icons so that I understood their purpose a little bit better
 - I merged some icons which looked similar
 - Therefore, I think before it's pushed to main, I'd like to test it a bit and make sure I know what each button is doing for coherency. Right now there's a bit of information lost in translation, and it shows.
 - I still think that even as it is right now it's a huge improvement. In particular, there weren't stock material icons for "culvert" or "split way" or "snap to existing nodes" which didn't feel contrived, so I made my own in inkscape. It feels very coherent now :D

I'll make a pull request for the draft and I'll have you take a look @simonpoole.
